aggregator/run_test.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Start aggregator itself and store its PID into variable
+/home/slabimic/Nemea/Nemea-Modules/aggregator/agg -i u:input_data,u:aggr_data -k SRC_IP -k DST_IP -s BYTES -s PACKETS -t a:60 & 
+PID=$!
+
+# Start logger to pickup aggregated data from unix socket to not to block the interface
+/usr/bin/nemea/logger -t -i u:aggr_data >/dev/null &
+
+# Start repeater with source data to send as input to aggregator
+/usr/bin/nemea/traffic_repeater -i f:/home/slabimic/Nemea_data/data.trapcap.*,u:input_data
```

Comment:
This is not a good way... We don'ŧ have `/home/slabimic/...` If you have some anonymized data, you can ship them with your patch instead of referencing non-existing file.

You can create a trapcap file or csv file that can be replayed and embed the data into the script. 
Please, see https://github.com/CESNET/Nemea-Modules/blob/master/report2idea/vportscan/test.sh and https://github.com/CESNET/Nemea-Modules/blob/master/report2idea/test.sh and for inspiration...